### PR TITLE
Newlogo

### PR DESCRIPTION
**Pull request by syedsimanta03**
Logo fixed

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import App, { AppProps } from 'next/app'
```
Comment (dmitrizagidulin) [inline pages/_app.tsx:1]: I suspect this is a typo, please fix.

File: components/ui/NavUserWidget/index.tsx [hunk 2]
```diff
@@ -23,12 +23,12 @@ type Props = {
 
 const mapStateToProps = (state: any) => {
   return {
-    auth: selectAuthState(state)
+    auth: selectAuthState(state),
```
Comment (dmitrizagidulin) [inline components/ui/NavUserWidget/index.tsx:26]:
```suggestion
    auth: selectAuthState(state)
```

File: components/ui/NavUserWidget/index.tsx [hunk 3]
```diff
@@ -23,12 +23,12 @@ type Props = {
...
   }
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  logoutUser: bindActionCreators(logoutUser, dispatch)
+  logoutUser: bindActionCreators(logoutUser, dispatch),
```
Comment (dmitrizagidulin) [inline components/ui/NavUserWidget/index.tsx:31]:
```suggestion
  logoutUser: bindActionCreators(logoutUser, dispatch)
```

File: pages/_app.tsx [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
+zimport App, { AppProps } from 'next/app'
```
Comment (dmitrizagidulin) [inline pages/_app.tsx:1]:
```suggestion
import App, { AppProps } from 'next/app'
```

**Comment by syedsimanta03** [comment]
I didn't write this code. Just need to push logo files

File: pages/_app.tsx (hunk 1 above)
Comment (syedsimanta03) [reply pages/_app.tsx:1]: Didn't write any code this time. Just the logos

**Comment by shawticus** [comment]
@syedsimanta03 can you resolve this merge conflict? (save the file somewhere, pull the latest, delete the file, replace it with your current, commit it, push it here) -- then we can approve the PR.

**Comment by syedsimanta03** [comment]
Sure


Sent from my Samsung Galaxy smartphone.
-------- Original message --------From: Shaw Walters <notifications@github.com> Date: 06/05/2020  1:04 p.m.  (GMT+06:00) To: xrchat/xrchat-client <xrchat-client@noreply.github.com> Cc: syedsimanta03 <syed.simanta06@gmail.com>, Mention <mention@noreply.github.com> Subject: Re: [xrchat/xrchat-client] Newlogo (#143) 

@syedsimanta03 can you resolve this merge conflict? (save the file somewhere, pull the latest, delete the file, replace it with your current, commit it, push it here) -- then we can approve the PR.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/xrchat/xrchat-client/pull/143#issuecomment-624478549",
"url": "https://github.com/xrchat/xrchat-client/pull/143#issuecomment-624478549",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by syedsimanta03** [comment]
Please check @shawticus https://github.com/xrchat/xrchat-client/pull/150

**Pull request closed without merging**